frontend/src/components/AccountReceiveTable.vue

```diff
@@ -78,6 +78,7 @@
         card-container-class="col q-col-gutter-md"
         :columns="mainTableColumns"
```

Comment:
*random placement for comment*

I'm seeing this weird behavior both on firefox and brave
<img width="821" alt="Screen Shot 2023-06-20 at 1 08 02 PM" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/a574e47b-23c8-441c-86e9-83f3c6a17a27">